narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/TxnTest.java

```diff
@@ -66,25 +62,26 @@ public static void tearDown() throws IOException {
 
     @Benchmark
     public void testTxn(Blackhole bh) throws IOException {
-        bh.consume(preExeTransaction());
-        // we want 2 txn participants so make 2 transactional requests
-        bh.consume(sendRequest());
-        bh.consume(TxnHelper.endTxn(txnClient, links));
+        bh.consume(testTxn());
     }
 
-    private boolean preExeTransaction() throws IOException {
```

Comment:
Please keep in mind I don't need the method 'preExeTransaction' anymore since the links variable is not global. In fact I need to make the assignment in a different way (line 69)